MANIFEST.in

```diff
@@ -4,7 +4,7 @@ recursive-exclude * *
 
 graft src/wagtail_factories
 
-include README.rst
+include README.md
```

Comment:
Converted the readme to markdown, I find it more pleasant to work with than RST, and I would like there to be less obstacles to getting some documentation written.